plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/BridgeVifDriver.java

```diff
@@ -206,8 +206,9 @@ public LibvirtVMDef.InterfaceDef plug(NicTO nic, String guestOsType, String nicA
         }
         String trafficLabel = nic.getName();
         Integer networkRateKBps = 0;
+        Integer bitsPerMbpsToKbps = 125;
         if (libvirtVersion > ((10 * 1000 + 10))) {
-            networkRateKBps = (nic.getNetworkRateMbps() != null && nic.getNetworkRateMbps().intValue() != -1) ? nic.getNetworkRateMbps().intValue() * 128 : 0;
+            networkRateKBps = (nic.getNetworkRateMbps() != null && nic.getNetworkRateMbps().intValue() != -1) ? nic.getNetworkRateMbps().intValue() * bitsPerMbpsToKbps : 0;
```

Comment:
As this piece of code is repeated several times, we could extract it to a common method (in VifDriverBase perhaps?)

Reply:
Good call. Updated the PR.